scripts/convert_dat.py

```diff
@@ -33,9 +37,22 @@
 
 FLOATS = NON_ZEROES + ZEROES
 
+def read_h5(
+    file_path: str
+):
+    logging.debug(f"Reading .zip file at {file_path}")
```

Comment:
This should be `info`. It's a good idea to get a print out of what file the program is working on regardless

Reply:
Realized the h5 function is unnecessary! There's another version of the data stored elsewhere on the machine.

Reply:
Awesome! That solves so many problems :)